scripts/build/mkversioninfo

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env sh
+set -eu
+
+: "${COMPANY_NAME=Docker, Inc.}"
```

Comment:
Can you set this to an empty string by default, so that it's an explicit configuration to set the company name?